fardel/apps/ecommerce/checkout/models.py

```diff
@@ -220,4 +221,36 @@ def dict(self):
             'total': self.get_total(),
             'item_price': self.get_price_per_item(),
             'is_shipping_required': self.is_shipping_required
-        }
\ No newline at end of file
+        }
+
+
+class Payment(db.Model):
+    __tablename__= "checkout_payments"
+
+    # order_id = db.Column(db.Integer, db.ForiegnKey('orders.id'))
+    id = db.Column(db.Integer, primary_key=True, index=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('auth_users.id'))
+    status_id = db.Column(db.Integer, default=0)    
+    """ 0: 'Waiting', 1: 'Done', 2: 'Failed' """
```

Comment:
In previous pull request i mentioned it that it needs to be converted to string for readability.